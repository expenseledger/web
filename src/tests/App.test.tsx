// import Enzyme, { shallow } from "enzyme";
// import Adapter from "enzyme-adapter-react-16";
// import toJson from "enzyme-to-json";
// import React from "react";
// import App from "../components/App";

// Enzyme.configure({ adapter: new Adapter() });

// describe("<App>", () => {
//     it("renders without crashing", () => {
//         const wrapper = shallow(<App />);
//         expect(toJson(wrapper)).toMatchSnapshot();
//     });
// });
