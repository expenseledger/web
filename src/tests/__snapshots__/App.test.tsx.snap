// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App> renders without crashing 1`] = `
<BrowserRouter>
  <Suspense
    fallback={<Loading />}
  >
    <Switch>
      <Route
        component={
          Object {
            "$$typeof": Symbol(react.lazy),
            "_init": [Function],
            "_payload": Object {
              "_result": [Function],
              "_status": -1,
            },
          }
        }
        exact={true}
        path="/signIn"
      />
      <AuthProtection>
        <lazy>
          <Route
            component={
              Object {
                "$$typeof": Symbol(react.lazy),
                "_init": [Function],
                "_payload": Object {
                  "_result": [Function],
                  "_status": -1,
                },
              }
            }
            exact={true}
            path="/"
          />
          <Route
            component={
              Object {
                "$$typeof": Symbol(react.lazy),
                "_init": [Function],
                "_payload": Object {
                  "_result": [Function],
                  "_status": -1,
                },
              }
            }
            exact={true}
            path="/wallet/:accountId/transactionList"
          />
          <Route
            component={
              Object {
                "$$typeof": Symbol(react.lazy),
                "_init": [Function],
                "_payload": Object {
                  "_result": [Function],
                  "_status": -1,
                },
              }
            }
            exact={true}
            path="/more"
          />
          <Route
            component={
              Object {
                "$$typeof": Symbol(react.lazy),
                "_init": [Function],
                "_payload": Object {
                  "_result": [Function],
                  "_status": -1,
                },
              }
            }
            exact={true}
            path="/category/setting"
          />
          <Route
            component={
              Object {
                "$$typeof": Symbol(react.lazy),
                "_init": [Function],
                "_payload": Object {
                  "_result": [Function],
                  "_status": -1,
                },
              }
            }
            exact={true}
            path="/wallet/setting"
          />
        </lazy>
      </AuthProtection>
    </Switch>
  </Suspense>
</BrowserRouter>
`;
